bin/lib/check_reqs.js

```diff
@@ -78,21 +79,46 @@ module.exports.check_ant = function() {
     });
 };
 
+module.exports.get_gradle_wrapper = function() {
+    var androidStudioPath;
+    if(os.platform() == "darwin") {
```

Comment:
A particular version of gradle, or just gradle in general?

Ubuntu gradle versions vary from release to release: http://packages.ubuntu.com/search?keywords=gradle
ArchLinux and homebrew (macOS) are both shipping the latest version (3.4)
Latest Android Studio ships 3.3